requirements/edx/github.txt

```diff
@@ -35,6 +35,7 @@ git+https://github.com/pmitros/pyfs.git@96e1922348bfe6d99201b9512a9ed946c87b7e0b
 -e git+https://github.com/edx-solutions/django-splash.git@7579d052afcf474ece1239153cffe1c89935bc4f#egg=django-splash
 -e git+https://github.com/edx/acid-block.git@e46f9cda8a03e121a00c7e347084d142d22ebfb7#egg=acid-xblock
 -e git+https://github.com/edx/edx-ora2.git@release-2015-05-07T14.06#egg=edx-ora2
+-e git+https://github.com/edx/edx-ora2.git@feanil/update_celery#egg=edx-ora2
```

Comment:
You have 2 versions of ora-2 here which is why your getting the "Double requirement given: edx-ora2" error on the builds.